jazz-ui/src/app/pages/service-overview/service-overview.component.ts

```diff
@@ -620,7 +620,7 @@ export class ServiceOverviewComponent implements OnInit {
         var k=2;
         this.sortEnvArr();
 
-        if(this.environ_arr!=undefined)
+        if(this.environ_arr!=undefined){
```

Comment:
indentation here and other places as well

Reply:
Done